doc/src/examples/ownable.md

```diff
@@ -0,0 +1,97 @@
+## Overview
+
+This example shows how you can use the implementation of
+[access-control](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/access/ownable) and
+[psp1155](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp1155) together to provide `only owner` rights to mint and burn tokens.
+
+## Steps
+
```

Comment:
Maybe we should add that you need to add dependancies for our own version of ink!
Because following these steps will not compile =)

Reply:
Yea, better to add usage of our version of ink=)